src/self-managed-debug/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "mz-self-managed-debug"
+description = "Debug tool for self-managed Materialize."
+version = "0.130.1"
```

Comment:
You probably want the current version here and elsewhere (right now it's `0.133.0-dev.0`, but you'll want to update it prior to merging.)

Also, to update it automatically, adjust line 41 and 61 in `bin/bump-version` so that the version gets updated automatically. The script takes care of calling `bin/bazel gen` to sync the bazel files.

Reply:
Acc ik Marta mentioned we'd most likely want to decouple the tool from LTS versions/materialize   https://materializeinc.slack.com/archives/C07PN7KSB0T/p1737567670003019?thread_ts=1737566344.547089&cid=C07PN7KSB0T, so we might want to make this just version 0.1.0. I think it's very doable that we can have the tool work for any version of materialize.

Reply:
If we want to include anything from `mz_introspection`, we'll need to version it the same way as the rest of Materialize. It's inherently unstable and not compatible across versions.

Reply:
sgtm!